src/wolfgame.py

```diff
@@ -3279,10 +3293,89 @@ def transition_day(cli, gameid=0):
         message.append("A chilling howl was heard last night. It appears there is another werewolf in our midst!")
         novictmsg = False
 
+    varkills = []
```

Comment:
Unnecessary, simply use bywolves instead. bywolves should **never** be removed from, so it can be used to see if wolves targeted someone or not. onlybywolves is used if you want to ensure that someone was ONLY targeted by wolves and nobody else.